### PR TITLE
CoinRanking Auth

### PR DESCRIPTION
**Pull request by micro-chipset**
- [ ] My submission is formatted according to the guidelines in the [contributing guide](/CONTRIBUTING.md)
- [ ] My addition is ordered alphabetically
- [ ] My submission has a useful description
- [ ] The description does not end with punctuation
- [ ] Each table column is padded with one space on either side
- [ ] I have searched the repository for any relevant issues or pull requests
- [ ] Any category I am creating has the minimum requirement of 3 items
- [ ] All changes have been [squashed][squash-link] into a single commit

[squash-link]: <https://github.com/todotxt/todo.txt-android/wiki/Squash-All-Commits-Related-to-a-Single-Issue-into-a-Single-Commit>

### CODE REVIEW

**Comment by micro-chipset** [comment]
Fix https://github.com/public-apis/public-apis/pull/1540

**Comment by micro-chipset** [comment]
Hello, @matheusfelipeog.
I have raised a PR against the label 'hacktoberfest' , waiting for feedback against the PR.

File: README.md [hunk 1]
```diff
@@ -297,7 +297,7 @@ API | Description | Auth | HTTPS | CORS |
 | [Coinlore](https://www.coinlore.com/cryptocurrency-data-api) | Cryptocurrencies prices, volume and more | No | Yes | Unknown |
 | [CoinMarketCap](https://coinmarketcap.com/api/) | Cryptocurrencies Prices | `apiKey` | Yes | Unknown |
 | [Coinpaprika](https://api.coinpaprika.com) | Cryptocurrencies prices, volume and more | No | Yes | Yes |
-| [CoinRanking](https://docs.coinranking.com/) | Live Cryptocurrency data | No | Yes | Unknown |
+| [CoinRanking](https://docs.coinranking.com/) | Live Cryptocurrency data | `apiKey` | Yes | Unknown |
```
Comment (matheusfelipeog) [inline README.md:300]: Can you update this link to `https://developers.coinranking.com/api/documentation`, please?

```suggestion
| [CoinRanking](https://developers.coinranking.com/api/documentation) | Live Cryptocurrency data | `apiKey` | Yes | Unknown |
```

**Review by matheusfelipeog — CHANGES_REQUESTED** [review]
Thanks for your contribution @micro-chipset ;)

Can you resolve this review,  please?

File: README.md (hunk 1 above)
Comment (micro-chipset) [reply README.md]: @matheusfelipeog fix

**Review by matheusfelipeog — APPROVED** [review]
Now it's Ok ✔

Thanks for your update <3

**Pull request merged**